build/yamls/antrea.yml

```diff
@@ -202,7 +202,7 @@ data:
     # - netdev
     # 'system' is the default value and corresponds to the kernel datapath. Use 'netdev' to run
     # OVS in userspace mode. Userspace mode requires the tun device driver to be available.
-    #ovsDatapathType: system
+    ovsDatapathType: system
```

Comment:
Why remove # here?

Reply:
That's related to your comment below, so I'll be replying there.